tower_modules/tower_job_template/tasks/main.yml

```diff
@@ -16,10 +16,10 @@
 
 - name: Wait for the project to be status=successful
   uri:
-    url: "{{ tower_host }}/api/v2/projects/{{ result.id }}/"
+    url: "{{ lookup('ENV', 'TOWER_HOST') }}/api/v2/projects/{{ result.id }}/"
     method: GET
-    user: "{{ tower_username }}"
-    password: "{{ tower_password }}"
+    user: "{{ lookup('ENV', 'TOWER_USERNAME') }}"
+    password: "{{ lookup('ENV', 'TOWER_PASSWORD') }}"
```

Comment:
@beeankha I am going to lose these tasks entirely when your change gets in. so that's why this winds up looking _worse_, it should be temporary.